demos/bank-of-anthos-asm-manifests/demo-manifests/frontend-custom-50-50.yaml

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# [START anthos-service-mesh_bank-of-anthos_virtualservice_frontend ]
+# [START servicemesh_demo_manifests_asm_gateway_gateway_bank_of_anthos_gateway]
```

Comment:
just going through, and I saw that these tags in this file are different~

Reply:
great catch!